src/smart-components/user/user.js

```diff
@@ -21,9 +24,45 @@ import { BAD_UUID, getDateFormat } from '../../helpers/shared/helpers';
 import { useIntl } from 'react-intl';
 import messages from '../../Messages';
 import './user.scss';
+import pathnames from '../../utilities/pathnames';
 
 let debouncedFetch;
 
+
+
+const UserDescription = ({user}) => {
+  const intl = useIntl();
+
+  return (
+    <Fragment>
+      {user?.orgAdmin ? (
+      <Fragment>
+        <span key="yes">
+          {intl.formatMessage(messages.orgAdministrator) + ": "}
+          <CheckIcon key="yes-icon" className="pf-u-ml-sm pf-u-mr-sm" />
+          {intl.formatMessage(messages.yes) }
+        </span>
+      </Fragment>
+    ) : (
+      <Fragment >
+        <span key="no" >{intl.formatMessage(messages.orgAdministrator) + ": "} 
+        <CloseIcon key="no-icon" className="pf-u-ml-sm pf-u-mr-sm" />
+        {intl.formatMessage(messages.no) }</span>
+      </Fragment>
+    )}
+      {user?.email && (
+        <TextContent className="rbac-page-header__description">
+         {typeof user?.email === 'string' ? <Text component={TextVariants.p}>{intl.formatMessage(messages.email)} : {user?.email}</Text> : user?.email}
```

Comment:
we have an extra space before `:`, 
also  usage of a template string could make the code more readable